c/discovery.c

```diff
@@ -148,7 +148,7 @@ static SoftwareInstance *addSoftware(DiscoveryContext *context,
   info->bestName = bestName;
   if ((context->ssctTraceLevel >= 1) ||
       (context->addressSpaceTraceLevel >= 1)){
-    printf("addSoftware info=0x%x type=0x%llx bestName=%s\n",info,info->type,info->bestName);
+    zowelog(NULL, LOG_COMP_DISCOVERY, ZOWE_LOG_INFO, "addSoftware info=0x%x type=0x%llx bestName=%s\n",info,info->type,info->bestName);
```

Comment:
Yeah, we could just leave it as it is. Looks like debug. Or since this file already has zowelog, we can use the debug level along with zowelog.

Reply:
@1000TurquoisePogs @ifakhrutdinov I agree with leaving it as it is, since that's what we've been doing with the rest of the trace level printf's. Just haven't gotten to reviewing this file yet.